management_api_app/main.py

```diff
@@ -27,10 +29,30 @@ def get_application() -> FastAPI:
     return application
 
 
+def initialize_logging(logging_level: int):
+    """
+    Adds the Application Insights handler for the root logger and sets the given logging level.
+
+    :param logging_level: The logging level to set e.g., logging.WARNING.
+    """
+    logger = logging.getLogger()
+    app_insights_instrumentation_key = os.getenv("APPINSIGHTS_INSTRUMENTATIONKEY")
```

Comment:
can we use the config instead - that picks up the env variables but then we have them all represented in one place

Reply:
Fixed